### DISCUSSION TITLE
Memory Expansion | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1008] [Q&A]
# Memory Expansion | Seeed Studio Wiki

Memory Expansion

https://wiki.seeedstudio.com/reComputer_J2021_J202_Flash_Jetpack/

**Comment by FyrbyAdditive** [comment]
Hopefully this information will save someone a lot of time!

For the J1020v2, if you want to follow official Nvidia tutorials to work the method via the command line re-installation above will not give you a properly working installation as the docker images will have trouble with missing host libraries, and you will only discover this when you get part the way into the jupyter notebooks. The fix for this is to use the command line SDK manager using the docker image containing Ubuntu 18.04 to install the latest Jetpack for this device, 4.6.4.

Remember to pass through privileged USB access to the docker container or it will not detect the Jetson. Also. there is not enough space in the eMMC to fit all the components. I select everything except the SDK components, as these take forever to flash to the eMMC and are quick on the SSD. Once those are installed transfer it to an NVMe disk using the "memory expansion" guide in this wiki, then use the SDK manager later to install the SDK components.

This also makes the nvidia docker commands work consistently with their instructions.
